### DISCUSSION TITLE
The generated password is duplicate

### DISCUSSION THREAD

**Discussion by top-player** [CodeDead/Advanced-PassGen#79] [General]
For example, when you choose to generate 101 two digit digital passwords, duplicate passwords will be generated.

**Comment by CodeDead** [comment]
Hi @top-player ,

Under the advanced option, you have the ability to disable duplicate passwords:
![image](https://github.com/CodeDead/Advanced-PassGen/assets/16293330/4f1e8151-d6f2-405d-8889-21a5885e6ef7)

If you uncheck this, it will not generate duplicate passwords.

**Reply by top-player** [reply]
oh i got it
